heareval/tasks/pipeline.py

```diff
@@ -171,7 +171,9 @@ def slugify_file_name(relative_path: str):
 
         The basic version here takes the filename and slugifies it.
         """
-        return f"{slugify(str(Path(relative_path).stem))}"
+        slug_text = str(Path(relative_path).stem)
+        slug_text = slug_text.replace("-", "_negative_")
+        return f"{slugify(slug_text)}"
 
     @staticmethod
     def get_stratify_key(df: DataFrame) -> Series:
```

Comment:
Where is stratify_key even used? I was searching the code and can't find it :|

Reply:
https://github.com/neuralaudio/hear-eval-kit/blob/4de804e091dab4c2e05385044b44aa9737b66d4c/heareval/tasks/util/luigi.py#L215

Reply:
Yes. In the above line, passing `none` in the stratify key, doesn't stratify and treats the key as not useful.

Reply:
That also needs a comment explaining